stdlib/public/Concurrency/AsyncStreamBuffer.swift

```diff
@@ -341,7 +340,7 @@ extension AsyncThrowingStream {
       }
     }
 
-    func cancel() {
+    @Sendable func cancel() {
```

Comment:
is that one necessary?

Reply:
I think so since it is used in the termination which is sendable